book/_toc.yml

```diff
@@ -61,6 +61,7 @@
         - file: tutorials/lidar/index
           sections:
             - file: tutorials/lidar/ICESat2_tutorial
+        - file: tutorials/machine-learning/Machine_Learning_Tutorial
```

Comment:
This does not show up correctly in the table of contents and is grouped under LIDAR.

We can merge this and as there is anyway a restructure with pull request #99 

Thoughts?
@aaarendt @scottyhq

Reply:
trying to run the notebook now on the hub, i'm uncomfortable adding tensorflow and keras to the default image this late in the game so seeing if we can pip install them at the top of the notebook